src/inference/src/dev/core_impl.cpp

```diff
@@ -802,8 +802,7 @@ ov::SoPtr<ov::ICompiledModel> ov::CoreImpl::compile_model_with_preprocess(ov::Pl
     std::shared_ptr<const ov::Model> preprocessed_model = model;
 
     // Disable conversion for proxy plugin and virtual devices to add pre-processing based on API of internal plugins
-    if (!is_new_api() && !std::dynamic_pointer_cast<InferenceEngine::IPluginWrapper>(plugin.m_ptr) &&
-        !is_virtual_device(plugin.get_name()) && !is_proxy_device(plugin)) {
+    if (!is_new_api() && !is_virtual_device(plugin.get_name()) && !is_proxy_device(plugin)) {
         ov::pass::Manager manager;
         manager.register_pass<ov::pass::AddPreprocessing>();
```

Comment:
maybe we need to remove API 1.0 preprocessing public API and then we could remove the whole `AddPreprocessing` pass?

Reply:
I expect a lot of failures in tests, so let's do it in next PR